opc/import_database_service_instance_test.go

```diff
@@ -0,0 +1,34 @@
+package opc
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccOPCDatabaseServiceInstance_importBasic(t *testing.T) {
+	resourceName := "opc_database_service_instance.test"
+
+	ri := acctest.RandInt()
+	config := testAccDatabaseServiceInstanceBasic(ri)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+		},
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckDatabaseServiceInstanceDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: config,
+			},
+			{
+				ResourceName:            resourceName,
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"parameter", "vm_public_key"},
+			},
```

Comment:
I don't believe this matters. We can't update anything about a service instance. Correct me if I'm wrong though